SWIG/inflation.i

```diff
@@ -679,7 +687,6 @@ class InterpolatedZeroInflationCurve : public ZeroInflationTermStructure {
                                    const Period& lag,
                                    Frequency frequency,
                                    bool indexIsInterpolated,
-                                   const Handle<YieldTermStructure>& yTS,
```

Comment:
I'd leave the deprecated version until it's removed from the library.  Unfortunately overloading will prevent kwargs from working, but I don't think there's a good way to do this...